dot/digest/digest.go

```diff
@@ -15,6 +17,8 @@ import (
 
 var (
 	_ services.Service = &Handler{}
+
+	ErrCannotDefineNextEpoch = errors.New("cannot define next epoch data and config")
```

Comment:
nit maybe rename to `ErrNextEpochDefinition`?

Reply:
Maybe `ErrDefineNextEpoch`? The `Cannot` part shouldn't be part of the error variable name I think.